src/config/env.ts

```diff
@@ -72,6 +72,8 @@ export interface EnvVars {
 	SQS_DISCOVERY_QUEUE_REGION: string;
 	SQS_DEPLOYMENT_QUEUE_URL: string;
 	SQS_DEPLOYMENT_QUEUE_REGION: string;
+	SQS_BRANCH_QUEUE_URL: string;
```

Comment:
You should put these 2 env vars in the 'required vars' array at the top of the file.

Reply:
good catch. done!